### ISSUE TITLE
Support for direct-to-camera rendering in URP

### ISSUE THREAD

**Issue by BitBarrel** [rive-app/rive-unity#33]
Currently, direct-to-camera rendering in Unity URP is not supported when using RiveScreen.cs. The build-in pipeline will be phased out eventually.

Starting points:

https://docs.unity3d.com/Packages/com.unity.render-pipelines.universal@14.0/manual/post-processing/post-processing-custom-effect-low-code.html

https://github.com/Cyanilux/URP_BlitRenderFeature

**Comment by pjarnfelt** [comment]
URP is working. Make sure to check the correct documentation and also the URP example project. Also, potentially search the issue list before posting. There are other posts about URP where this confusion has been addressed.

**Comment by BitBarrel** [comment]
Thank you for your answer. I used the search function in both google and on this github page and nothing showed up. The documentation page on the rive-unity github. Both  https://help.rive.app and https://rive.app/community/ do not give any search result when typing in URP. 

Would you be so kind to point me to the correct documentation page?

**Comment by pjarnfelt** [comment]
Yeah. The documentation is a bit of a mess. They're also moving things to a new site it seems, and a lot of the discussion was on discord which they deleted. But here is an example of rive working on URP.  https://github.com/rive-app/rive-unity-examples/tree/main/minion-head-tracking-urp
In the URP method you cannot use a command buffer on the camera, but must rather call submit and advance on the renderer in the update method
`private void Update()
    {
        m_riveRenderer.Submit();
        if (m_stateMachine != null)
        {
            m_stateMachine.Advance(Time.deltaTime);
        }
    }`

**Comment by BitBarrel** [comment]
Thanks, much appreciated. 

I re-worded the issue post a bit to make it more clear that it is about direct-to-camera rendering.

**Comment by damzobridge** [comment]
This should be addressed if you're using the new RivePanel and RiveWidget components which are available as of [v0.3.2](https://github.com/rive-app/rive-unity/releases/tag/v0.3.2).

If you don't want to display the Rive graphic within uGUI or UIToolkit, you can display the RenderTexture created by a RivePanel directly to the screen in any way that fits your project (e.g. using the approach in RiveScreen).
